Alicerce/Sources/Logging/Chronicle/Log.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 
 final class Log {
     
-    fileprivate static var providers = Array<ProviderProtocol>()
+    fileprivate static var providers = Array<LogProvider>()
```

Comment:
This could be `private`, no? 🤐

what do you think of [LogProvider]()

I also did a comment regarding the naming, we could nest the protocol inside the `Log` class